### ISSUE TITLE
4 features that I am missing in v4.x

### ISSUE THREAD

**Issue by ralxxx** [y20k/transistor#279]
Thanks for all the work rewriting the transistor app. Just got it via F-Droid this morning. I'd like to share with you my first thoughts, but I am not too sure where to do that. I decided using the Feature Request section (not sure whether this is appropriate). 

I am really, really glad for being able to add easily a new radio station, that makes things really a lot easier. The apps works smoothly, so far I did not came across any bugs.  

But...I am not sure yet, if I am so happy with most the other changes.  
In the past I used Transistor almost daily as my main radio player - with the same set of some 30 radio stations. What I enjoyed with transistor was that it focused on core functionality, that the layout was simple and calm. And it was very difficult to make a mistake (such as deleting a station by mistake). Although normally I am happy with news features in apps, transistor was an app that for me was close to being perfect, I loved it for its bare and minimalistic design. 

Hereafter what I preferred from the previous versions (my feature proposals for this version). 

- **Deleting radio stations.** With an app such as Transistor, this is rather something that needs to be done rather rarely. But it is too easy now to swipe away a station (and then it is gone). The risk that it is done in an unwanted manner is too big. My feature proposal would be: Go back to how it was before. 

- **General look and feel 1.** The fact that each and every radio station has a play and a stop button and the three little dots creates a lot of 'unrest' in terms of appearance. Selecting a station and starting with the central start/stop button or shortcutting with a 'long press' on the station itself, as it was with the previous version, was perfect. My feature proposal would be: Go back to how it was before. 

- **General look and feel 2.** The spacing between the stations does not increase clarity.
My feature proposal would be: Decrease spacing (go back to how it was before). 

- **Swiping down - "Look for changes".** This is not a feature that it being needed on an everyday basis, therefore it would make make sense to me to hide it in the settings that can be reached via the little settings wheel next to the "add new station button".

**Comment by y20k** [comment]
Hi @ralxxx 

Thanks for your feedback. My comments:

> Swiping down - "Look for changes"

I agree. This feature will move into settings.

> General look and feel

My plan is to introduce an Edit station dialog or screen, which would host all feature in the three dots menu. I need to see how the user interaction will work in practice when this feature is present. Maybe the swipe to delete feature that you don't like will move there.

> every radio station has a play button

I always felt that the previous solution involved unnecessary taps and finger movement. And it seemed an unusual approach compared to other media players. The new version adds visual complexity but I think that overall it is an improvement.

> spacing between the stations does not increase clarity

You are probably right. But I think I will wait a bit before making changes here. I want to wait to see how future changes will impact the user interaction - For example the mentioned Edit screen, or the possible move of the delete feature.

Anyway thanks for your thoughts!

**Comment by ralxxx** [comment]
Just a little amendment to my comments. I senior member of my family is using Transistor to listen to radio. Probably not on purpose he swiped away most of his radio stations. My conclusion: In my opinion Transistor serves as radio player.Deleting stations happens rather rarely, therefore the deleting feature should be rather hidden.

**Comment by Pentaphon** [comment]
@y20k Just a quick question since this issue is not too specific: is the latest Transistor 4.0.x using Exoplayer 2.11.7?

**Comment by y20k** [comment]
Yep. It currently uses v2.11.7 (see https://github.com/y20k/transistor/blob/master/app/build.gradle)

**Comment by cbiere** [comment]
I agree with ralxxx. In my humble opinion, the app would be more sensible and accessible with an interface like Google Contacts. The current interface is okay for its minimalistic approach.
